### DISCUSSION TITLE
对图片功能的疑问

### DISCUSSION THREAD

**Discussion by pupupuxi** [WatsonYao/nian_app#125] [Q&A] [answered]
图片的复制和剪切有不方便的地方。之前有人的建议是隐藏相册，但是我隐藏相册之后，打开nian有显示不出来图片的情况。
请问是否有什么更好的方法可以解决问题呢？或者支持插入图片链接，读取路径之类的？

**Comment by WatsonYao** [comment, marked as answer]
单机应用肯定需要图片在手机上某个位置的，这个位置也不能隐藏到应用无法识别。
不清楚其他人如何隐藏的，这边红米手机在相册里面选择屏蔽nian文件夹就可以了，也不影响nian里面图片的展示。
